source/iface/device.rst

```diff
@@ -195,97 +297,987 @@ Device Info
++---------------------------------------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::platform``
+
+Returns the SYCL platform associated with this SYCL device.
+
++--------------------------------------+
+| Return type: ``sycl::platform``      |
++--------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::name``
+
+Returns the device name of this SYCL device.
+
++--------------------------------------+
+| Return type: ``std::string``         |
++--------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::vendor``
+
+Returns the vendor of this SYCL device.
+
++--------------------------------------+
+| Return type: ``std::string``         |
++--------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::driver_version``
+
+Returns a vendor-defined string describing the
+version of the underlying backend software driver.
+
++--------------------------------------+
+| Return type: ``std::string``         |
++--------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::profile``
+
+Deprecated in SYCL 2020.
+
+Returns a vendor-defined string describing the
+version of the underlying backend software driver.
+
+Only supported when using the OpenCL backend.
+Throws an exception with the ``errc::invalid`` error code if
+used with a device whose backend is not OpenCL.
+
+The value returned can be one of the following strings:
+``FULL_PROFILE``, ``EMBEDDED_PROFILE``.
+
++--------------------------------------+
+| Return type: ``std::string``         |
++--------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::version``
+
+Returns a backend-defined device version.
+
++--------------------------------------+
+| Return type: ``std::string``         |
++--------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::backend_version``
+
+Returns a string describing the version of the SYCL backend
+associated with the device.
+
+The possible values are specified in the SYCL backend
+specification of the SYCL backend associated with the device.
+
++--------------------------------------+
+| Return type: ``std::string``         |
++--------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::aspects``
+
+Returns a ``std::vector`` of :ref:`aspect <device-aspects>`
+values supported by this SYCL device.
+
++----------------------------------------------------+
+| Return type: ``std::vector<sycl::aspect>``         |
++----------------------------------------------------+
+
+
+.. rubric:: ``sycl::info::device::extensions``
+
+Deprecated, use ``sycl::info::device::aspects`` instead.
+
+Returns a ``std::vector`` of extension names
+(the extension names do not contain any spaces) supported by
+this SYCL device.
+
+The extension names returned can be vendor supported extension names
+and one or more of the following Khronos approved extension names.
```

Comment:
The sentence above implies there will be a list of "Khronos approved extension names", but there is no such list.

Since this is a deprecated query, I think it would be OK to just say "The extension names returned can be vendor supported extension names", without listing the Khronos ones.